doc/source/missing_data.rst

```diff
@@ -242,7 +228,7 @@ with missing data.
 Filling missing values: fillna
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The **fillna** function can "fill in" NA values with non-NA data in a couple
+The :meth:`~DataFrame.fillna` function can "fill in" NA values with non-NA data in a couple
```

Comment:
Instead of "The :meth:\`method\` function", write it as just ":meth:\`method\`". From the context it's clear that it's a function / method.